Makefile

```diff
@@ -98,13 +94,25 @@ vendor:
 	cp -rf --no-preserve=mode "$${_src}" "$${_dst}"
 .PHONY: vendor
 
+# GENERATE_YAML_ENV_VARS is the list of all environment variables expected by generate-yaml.sh
+# to build example manifests for clusterctl. The list of environment variables here should contain
+# the environment variables expected in generate-yaml.sh
+GENERATE_YAML_ENV_VARS = -e CLUSTER_NAME -e SERVICE_CIDR -e CLUSTER_CIDR -e CAPV_YAML_VALIDATION \
```

Comment:
Hi @andrewsykim,

Please use `:=` when assigning variables in Make that do not ever need to be [recursively evaluated](https://www.gnu.org/software/make/manual/html_node/Setting.html).